daml-lf/interpreter/BUILD.bazel

```diff
@@ -7,9 +7,12 @@ load(
     "da_scala_test",
     "da_scala_test_suite",
     "lf_scalacopts",
+    "lf_scalacopts_stricter",
 )
 load("@io_bazel_rules_scala//scala:scala.bzl", "scala_repl")
 
+print(lf_scalacopts_stricter)
```

Comment:
@nickchapman-da Did you mean to leave this in? I opened https://github.com/digital-asset/daml/pull/11091, if this is not supposed to be in you only need to approved, otherwise feel free to close it. 🙂